doc/src/manual/profile.md

```diff
@@ -37,17 +37,17 @@ any alternatives.
 Let's work with a simple test case:
 
 ```julia
-function myfunc()
-    A = rand(100, 100, 200)
-    maximum(A)
-end
+julia> function myfunc()
+           A = rand(100, 100, 200)
+           maximum(A)
+       end
 ```
 
 It's a good idea to first run the code you intend to profile at least once (unless you want to
 profile Julia's JIT-compiler):
 
 ```julia
-julia> myfunc()  # run once to force compilation
+julia> myfunc() # run once to force compilation
 ```
 
 Now we're ready to profile this function:
```

Comment:
Yeah, the output from the profile is quite different now. Someone with knowledge how things are started up should update that section.